Userland/Libraries/LibIPC/Encoder.cpp

```diff
@@ -106,6 +106,16 @@ Encoder& Encoder::operator<<(float value)
     return *this << u.as_u32;
 }
 
+Encoder& Encoder::operator<<(double value)
+{
+    union bits {
+        double as_double;
+        u64 as_u64;
+    } u;
+    u.as_double = value;
+    return *this << u.as_u64;
```

Comment:
Isn't this [UB in C++](https://stackoverflow.com/questions/11373203/accessing-inactive-union-member-and-undefined-behavior)?

Reply:
It doesn't appear to be, as the float encoder does the same thing with u32 and I "copied" that. It also seems to me like that's what unions are for.

Reply:
You could use `AK::bit_cast`

Reply:
Then I should rewrite the float code as well, if that's the better option.